var/spack/repos/builtin/packages/intel-oneapi-compilers/package.py

```diff
@@ -20,6 +20,14 @@
             "url": "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/69f79888-2d6c-4b20-999e-e99d72af68d4/intel-fortran-compiler-2025.0.0.723_offline.sh",
             "sha256": "2be6d607ce84f35921228595b118fbc516d28587cbc4e6dcf6b7219e5cd1a9a9",
         },
+        "nvidia-plugin": {
+            "url": "https://developer.codeplay.com/api/v1/products/download?product=oneapi&variant=nvidia&version=2025.0.0&filters[]=12.0&filters[]=linux",
+            "sha256": "e878a6403da046ad030b5f8df299ac866b1a11aa",
+        },
+        "amd-plugin": {
+            "url": "https://developer.codeplay.com/api/v1/products/download?product=oneapi&variant=amd&version=2025.0.0&filters[]=6.1.0&filters[]=linux",
+            "sha256": "8da98029c431495ad787528eadb4dbb5d22daac6",
```

Comment:
```suggestion
            "sha256": "2c5a147e82f0e995b9c0457b53967cc066d5741d675cb64cb9eba8e3c791a064",
```

Looks like you used `shasum` instead of `sha256sum`